plugins/sharding/stream_shard_executor.rb

```diff
@@ -91,11 +91,15 @@ def ensure_prepared
         @expression_builder = RangeExpressionBuilder.new(@shard_key,
                                                          @target_range)
         @expression_builder.filter = @filter
-        index_info = @shard_key.find_index(Operator::LESS)
-        if index_info
-          index = index_info.index
-          @context.referred_objects << index
-          @range_index = index if use_range_index?
+        if use_range_index?
+          index_info = @shard_key.find_index(Operator::LESS)
+          if index_info
+            index = index_info.index
+            @context.referred_objects << index
+            @range_index = index
+          else
+            decide_use_range_index(false, "no range index", __LINE__, __method__)
```

Comment:
`decide use_range index` returns inputted `use` but we don't need it at this line.
It may be better to create `log_use_range index` and use it.

Reply:
No problem. We don't need `log_use_range_index`.

Reply:
This code removed because I adopted https://github.com/groonga/groonga/pull/1349#discussion_r840165682 approach.